config/settings.defaults.coffee

```diff
@@ -11,11 +11,11 @@ settings =
 		# Choices are
 		# s3 - Amazon S3
 		# fs - local filesystem
-		if process.env['AWS_KEY']? or process.env['S3_BUCKET_CREDENTIALS']?
```

Comment:
This is backwards incompatible, would you mind adding a fallback on the old environment variable?

Reply:
Yes, that's a wise idea.